src/config.h.in

```diff
@@ -1,470 +0,0 @@
-/* src/config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
We're not checking in to git any of the outputs from `./autogen.sh` - except this one? I'm not sure why. We do check in config.h.generic, which is reasonable (to me).

Reply:
This is a very good question. This file always changes.

Reply:
I guess it's just a historical. It's been in the repo since 2015 - Philip had previous removed it in 2014, then checked it back in again.

Philip added it back alongside `src/config.h`, which was subsequently removed (again). But `config.h.in` was left.

I think it should just go. We have `config.h.generic` for users who want to work with an alternative build system to autoconf.

Reply:
I might be wrong, but I think the idea was that then you can base your build in this file without having to run auto tools at all.

Agree that is somehow redundant with `config.h.generic`, but it is because "generic" is a derivative from it (through a Makefile rule)